src/pages/FridgeDetailPage.js

```diff
@@ -1 +1,223 @@
-export const FridgeDetailPage = () => null;
+/* eslint-disable react/forbid-prop-types */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { Text, View } from 'react-native';
+import * as Animatable from 'react-native-animatable';
+
+import { DateRangeSelector } from '../widgets/DateRangeSelector';
+import {
+  CardDetail,
+  ColdBreachIcon,
+  DataTablePageView,
+  FlexRow,
+  HotBreachIcon,
+  PaperSection,
+  SpacedChildren,
+} from '../widgets';
+import { VaccineChart } from '../widgets/VaccineChart';
+import { NoBreachMan } from '../widgets/NoBreachMan';
+import { FridgeActions } from '../actions/FridgeActions';
+import { AfterInteractions } from '../widgets/AfterInteractions';
+
+import {
+  selectAverageTemperature,
+  selectBreaches,
+  selectColdCumulativeBreach,
+  selectHotCumulativeBreach,
+  selectLeastRecentTemperatureLogDate,
+  selectMinAndMaxDomains,
+  selectMinAndMaxLogs,
+  selectMostRecentTemperatureLogDate,
+  selectNumberOfColdConsecutiveBreaches,
+  selectNumberOfHotConsecutiveBreaches,
+  selectTemperatureLogsFromDate,
+  selectTemperatureLogsToDate,
+} from '../selectors/fridge';
```

Comment:
That's a lot of selectors.

I think as part of performance tuning (as it is a bit slow), looking at putting a wrapper around the chart which is connected to redux directly, as well as for the cards at the bottom, so the picking of state fields can be separated out into each of the two parts to be concerned about